static/js/components/ArticleEditor.js

```diff
@@ -53,12 +54,15 @@ export default class ArticleEditor extends React.Component<Props> {
       })
 
       if (!readOnly) {
-        editor.model.document.on("change:data", event => {
-          if (onChange) {
-            onChange(event, editor)
```

Comment:
Could you explain why this line was dropped as part of these changes? It seems unrelated.

Reply:
it was mistakenly inserted in the first place, so when I noticed it I just took it out